### PR TITLE
Update version on README

### PR DESCRIPTION
**Pull request by akeller**
Small changes to the README, update to the latest version of the client.

### CODE REVIEW

File: go/go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/camunda/camunda-platform-get-started/go
 
-go 1.17
+go 1.20
```
Comment (akeller) [inline go/go.mod:3]: 1.20 is the latest version, but I don't know if we wanted to bump the minimum pre-requisite.

Comment (abbasadel) [reply go/go.mod:3]: Sorry. I just noticed that. 
I would suggest to go for 1.19 as this is what we use in [zeebe go client](https://github.com/camunda/zeebe/blob/main/clients/go/go.mod#L3)

Comment (akeller) [reply go/go.mod:3]:
```suggestion
go 1.19
```

**Comment by akeller** [comment]
Stale. Closing.

**Pull request closed without merging**